packages/memoize/tsconfig.json

```diff
@@ -14,5 +14,5 @@
     "typeRoots": ["../"],
```

Comment:
I wouldn't move this file into the root of the `memoize` package. This `tsconfig.json` is intended for the type tests in the `types` directory. The `packages/memoize/src` should be controlled by the very root `tsconfig.json` and not the `packages/memoize/tsconfig.json`

Reply:
Ah gotcha! I'm used to composite projects is why I did that, I'll move it back, thanks!

Reply:
Done in c31cc38